app/components/AnalyzeComponent.js

```diff
@@ -32,60 +30,35 @@ interface Props {
   jupyterActions: Object;
 }
 
-interface State {
-  eegFilePaths: Array<?{
-    key: string,
-    text: string,
-    value: { name: string, dir: string }
-  }>;
-  selectedFilePaths: Array<?string>;
-}
-
 export default class Analyze extends Component<Props, State> {
   props: Props;
-  state: State;
   handleDropdownChange: (Object, Object) => void;
   handleLoadData: () => void;
+  handleAnalyze: () => void;
   handleChannelDropdownChange: (Object, Object) => void;
 
   constructor(props: Props) {
     super(props);
-    this.state = {
-      eegFilePaths: [{ key: "", text: "", value: { name: "", dir: "" } }],
-      selectedFilePaths: []
-    };
-    this.handleDropdownChange = this.handleDropdownChange.bind(this);
-    this.handleLoadData = this.handleLoadData.bind(this);
+    this.handleAnalyze = this.handleAnalyze.bind(this);
     this.handleChannelDropdownChange = this.handleChannelDropdownChange.bind(
       this
     );
   }
 
   componentWillMount() {
     if (isNil(this.props.kernel)) {
-      this.props.jupyterActions.launchKernel();
+      this.props.history.push("/clean");
     }
-    this.setState({
-      eegFilePaths: readEEGDataDir(this.props.type).map(filepath => ({
-        key: filepath.name,
-        text: filepath.name,
-        value: filepath
-      }))
-    });
   }
 
-  handleDropdownChange(e: Object, props: Object) {
-    this.setState({ selectedFilePaths: props.value });
+  handleAnalyze() {
+    this.props.jupyterActions.loadPSD();
```

Comment:
Let's add loadERP here too and make the loadERP epic not listen to SET_PSD anymore

Reply:
done